### ISSUE TITLE
[UI5-Textarea - show exceeded text  | UI5 Web Component -react ]: "show exceeded text" should have aria-live region so that screen reader users are updated with the remaining character count

### ISSUE THREAD

**Issue by kushalkaranjkar** [SAP/ui5-webcomponents#6744]
### Bug Description

On the UI5 Text Area control - The character count label should be an aria-live region so that screen reader users are updated with the remaining character count as when update happens.
aria-live="polite" can be used so that it is not disruptive other announcements.
Accessibility team from SF found this issue on the textareas and suggested above enhancement to "show Exceeded text" region. 

Actual Result : 
As count of "characters remaining" text not announced by screen reader

### Affected Component

TextArea

### Expected Behaviour

Expected Results:
Any visible change in content/context that is triggered as a result of any action, must be announced to screen reader users

1. aria-live="polite" should be added to region.

### Steps to Reproduce

1. Create TextArea with show-exceeded-text attribute and some max character limit
![image](https://user-images.githubusercontent.com/845448/226491639-4678b9c8-4973-4e25-93f0-f1616514543e.png)
2. Run JAWS or other screen reader
3. Type characters inside TextArea
4. Screen readers do not announced "remaining character" as user typed.

### Priority

Medium

### UI5 Web Components Version

v1.10.5

### Browser

Chrome

### Operating System

windows 10

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by s-todorova** [comment]
Hello @SAP/ui5-webcomponents-topic-rl,

You can reproduce the issue on the [TextArea test page](https://sap.github.io/ui5-webcomponents/main/playground/main/pages/TextArea/) and JAWS: When typing into a TextArea with a character count limit, the SR doesn't announce the remaining characters as you continue to type.
From my testing, the issue is reproducible with JAWS,  but not with NVDA (2022.2.3).

Could you please take a look at this?

Best regards,
Siyana

**Comment by elenastoyanovaa** [comment]
Hello @kushalkaranjkar, 

Announcing the remaining characters on each keystroke may annoy the users and this is why this is not part of the core accessibility features and the components. As you can check this is also not available for the sap.m.TextArea control. 

Even though announcement may be very helpful for the users, if you pollute the speech output with various announcements this might confuse the user, instead of helping him. The mentioned feature is not part of our accessibility specification.

Additionally, we provide extra API's for applications to enrich the existing accessibility of the web components to fit their use case and I could suggest one for you. Please check the Invisible Messaging section in the [Accessibility Documentation](https://sap.github.io/ui5-webcomponents/playground/advanced/accessibility/).
You can use it to announce the remaining characters on the input event of the textarea. Please check the example usage linked in the documentation.

Kind Regards.
Elena
